variables.tf

```diff
@@ -40,3 +40,18 @@ variable "emr_instance_type" {
     production  = "m5.2xlarge"
   }
 }
+
+variable "metadata_store_master_username" {
+  description = "Username for metadata store master RDS user"
+  default     = "hive"
+}
+
+variable "metadata_store_pdm_reader_username" {
+  description = "Username for metadata store readonly RDS user"
+  default     = "pdm-reader"
```

Comment:
I thought currently we have no requirement for a PDM-READER role, only writer? curretly pdm-reader does not exist.